RetailCoder.VBE/UI/Controls/NumberPicker.xaml.cs

```diff
@@ -14,30 +14,17 @@ public partial class NumberPicker : IDataErrorInfo
 
         public int NumValue
         {
-            get
-            {
-                return (int)GetValue(NumValueProperty);
-            }
+            get => (int)GetValue(NumValueProperty);
```

Comment:
Not a deal-breaker, but I would use braces for the get if you are going to use them for the set...